[#8] Update README with latest Ui mockup and project details (#13)
AY2223S1-CS2103T-T15-2/tp: README.md

# Class-ify

[![codecov](https://codecov.io/gh/AY2223S1-CS2103T-T15-2/tp/branch/master/graph/badge.svg?token=LKPSW2V6NG)](https://codecov.io/gh/AY2223S1-CS2103T-T15-2/tp)

![Ui](docs/images/Ui.png)

## Description

> Class-ify is a class management application build specifically for teachers to handle all administrative related matters.

Reviewer: ```suggestion
Class-ify is a class management application built specifically for teachers to handle all administrative-related matters.
```

Author: Applied suggested changes in latest commit

> The Class-ify is an all-in-one platform to view and organize your students’ details, and categorize them accordingly.

Reviewer: ```suggestion
It is an all-in-one platform to view and organize your students’ details, and categorize them accordingly.
```
